Lib/test/test_warnings/__init__.py

```diff
@@ -877,6 +877,24 @@ def test_showwarning(self):
                                 file_object, expected_file_line)
         self.assertEqual(expect, file_object.getvalue())
 
+    def test_formatwarning_override(self):
+        # bpo-35178: Test custom formatwarning can receive line as positional
```

Comment:
It isn't obvious that the last parameter of myformatwarning() is usually called line, but here it's called "text" to test that the argument is passed as a position argument and not as a keyword argument. Try to rephrase the comment.

Reply:
How about indicating line could be keyword or positional argument inferring `line=line` like "Test custom formatwarning can receive line as keyword or positional argument" ? Since line is a common word and also indicates the warning line it's little difficult for me to convey this. Any suggestions would be helpful. Thanks.

Reply:
Something like this?

```
# bpo-35178: Test that a custom formatwarning function gets
# the 'line' argument as a positional argument,
# and not as a keyword argument
```

(I let you format it properly)

Reply:
I would make a minor edit of "not only as a keyword argument" since this reads a little like line as a keyword argument is not allowed. How about below ? 

```
bpo-35178: Test that a custom formatwarning function gets the 'line' 
argument as a positional argument, and not only as a keyword argument
```

Reply:
Yeah, that's more explicit than the current comment ;-)

Reply:
Rephrased the comment as noted.